src/injected-js/setupCustomViewEventAssassin.ts

```diff
@@ -43,7 +42,7 @@ function shouldBlockEvent(event: KeyboardEvent): boolean {
 
   // Block all escape key presses inside a custom view, even when an input
   // is focused.
-  if (event.key === 'Escape' && closest(target, '.inboxsdk__custom_view')) {
+  if (event.key === 'Escape' && target.closest('.inboxsdk__custom_view')) {
```

Comment:
Above here, we have a type assertion for `target` to be a  `HTMLElement`. To avoid a potential `TypeError`, we should remove that type assertion, and check in this if block for something like:

```suggestion
  const target = event.target;
  // ...
  if (event.key === 'Escape' && target instanceof HTMLElement && target.closest('.inboxsdk__custom_view')) {
```